packages/plugin-essentials/sources/index.ts

```diff
@@ -74,6 +75,7 @@ const plugin: Plugin = {
     remove,
     run,
     up,
+    whoami,
```

Comment:
This command would definitely not be part of the `plugin-essentials` (it's only meant to contain very generic commands that would make sense to any package manager, not only JS).

As we discussed I think the best is to create a `plugin-npm-cli` package that would contain this and could be extended to contain the other npm-specific commands.

Reply:
I put a very basic skull for the npm cli plugin in #123 (since I also need it to implement publish I figured it would prevent merge conflicts do to it in a separate PR) 🙂